business/checkers/authorization/no_host_checker.go

```diff
@@ -16,7 +17,6 @@ type NoHostChecker struct {
 	Namespace           string
 	Namespaces          models.Namespaces
```

Comment:
Another thought: if we assume that the list of services will contain all populations from all namespaces, do we need this list of namespaces?